packages/create-cloudflare/templates-experimental/next/c3.ts

```diff
@@ -18,7 +18,14 @@ const generate = async (ctx: C3Context) => {
 };
 
 const configure = async () => {
-	const packages = ["@opennextjs/cloudflare", "@cloudflare/workers-types"];
+	const packages = [
+		// Note: this is using the experimental pkg.pr.new prerelease
+		//       we do this because the stable @opennextjs/cloudflare package
+		//       does not yet support Next 15, we should move away from this
+		//       as soon as possible
+		"https://pkg.pr.new/@opennextjs/cloudflare@experimental",
```

Comment:
Haven't looked at the details but the experimental branch should have an open next config file and a different wrangler.toml.
Should they be updated as part of the PR?

Also loading instrumentation will fall on 15.0.3 if not patched.

Reply:
ah yeah of course, good callout 😓👍